src/packages/version-check/package.json

```
@@ -0,0 +1,66 @@
+{
+  "name": "@ganache/version-check",
+  "publishConfig": {
+    "access": "public"
+  },
+  "version": "0.1.0",
+  "description": "",
+  "author": "David Murdoch",
+  "homepage": "https://github.com/trufflesuite/ganache/tree/develop/src/packages/version-check#readme",
+  "license": "MIT",
+  "main": "lib/index.js",
+  "typings": "typings",
+  "source": "index.ts",
+  "directories": {
+    "lib": "lib",
+    "test": "tests"
+  },
+  "files": [
+    "lib",
+    "typings"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/trufflesuite/ganache.git",
+    "directory": "src/packages/version-check"
+  },
+  "scripts": {
+    "tsc": "tsc --build",
+    "test": "nyc npm run mocha",
+    "test:report": "nyc --reporter=html npm run mocha",
```

It'd be cool to update all our code (in a different PR, of course) to always generate both a text report (shown on the CLI) and the HTML report. This works:

```suggestion
    "test": "nyc --reporter lcov --reporter text npm run mocha",
```

p.s., we use `lcov` instead of `html` in some other ganache packages; I don't know why or if it is different than using `html`.

---

`lcov` calls `html`

---

Since this would be the only place in our codebase where we use the `html` reporter, and it is the same thing as lcov, I think it makes sense to align with the others.